spec/forms/idv/phone_form_spec.rb

```diff
@@ -69,7 +69,7 @@
         let(:optional_params) { { delivery_methods: [:sms] } }
 
         it 'uses the user phone number as the initial phone value' do
-          expect(subject.phone).to eq('+1 787 234 5678')
+          expect(subject.phone).to eq('+1 787-234-5678')
```

Comment:
Interested by the formatting changes; this won't cause any downstream issues will it?

Reply:
This occurred to me too, and I didn't think it would. Phone fingerprints should be using a standard format (e164) and any comparisons format it  prior to evaluating like [here](https://github.com/18F/identity-idp/blob/5ba3ded47ee1bb997a0caf62263accfa6fffcad0/app/forms/new_phone_form.rb#L120-L122)